codegen/generators/server/generator.go

```diff
@@ -0,0 +1,30 @@
+/*
```

Comment:
yes safe to remove. It's been replaced by https://github.com/goadesign/goa/blob/v2/codegen/generators/server.go